recipes/calculix/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "calculix" %}
+{% set version = "2.14" %}
+{% set sha256 = "59de9965d8f69141a34519101f7dd932c7d8d2be15480b31062b92512545fec4" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}_{{ version }}.tar.bz2
+    url: http://www.dhondt.de/ccx_{{ version }}.src.tar.bz2
+    sha256: {{ sha256 }}
+    patches:
+        - deps.patch
+        - pthread.patch
+        - u_free.patch
+        - mingw.patch  # [win]
+
+build:
+    number: 0
+
+requirements:
+    build:
+        - {{ compiler("c") }}  # [unix]
+        - {{ compiler('m2w64_c') }}  # [win]
+        - {{ compiler("fortran") }}  # [unix]
+        - {{ compiler('m2w64_fortran') }}  # [win]
+        - perl
+    host:
+        - arpack
+        - spooles
+        - openblas
+        - m2w64-openblas  # [win]
+
+test:
+    commands:
+        - test -f ${PREFIX}/bin/ccx  # [unix]
+        - test -f %LIBRARY_PREFIX%\bin\ccx.exe  # [win]
+
+
+about:
+    home: http://www.dhondt.de/
+    license: GPL 2
+    license_family: GPL
+    # license_file: COPYING
```

Comment:
GPL code requires the license.  You may need to put the license in the recipe.  See https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Reply:
thanks, done.

Reply:
also contacted the author to apply patches and add license file for next version.